js/web/lib/wasm/jsep/backend-webgpu.ts

```diff
@@ -361,26 +360,19 @@ export class WebGpuBackend {
           case 4:
             baseAlignment = 16;
             break;
-          case 5:
-            baseAlignment = 16;
-            break;
-          case 6:
-            baseAlignment = 16;
-            break;
           default:
-            throw new Error(`unsupported data length: ${data.length}`);
+            baseAlignment = 16;
         }
 
-        if (preLength === 5 || preLength === 6) {
-          baseAlignment = 16;
-        }
         if (baseAlignment > maxAlignmentOfField) {
           maxAlignmentOfField = baseAlignment;
```

Comment:
If I remember correctly, `maxAlignmentOfField` was used for `struct` by 5d/6d in tfjs. Since we used array in ort, this variable can be removed safely. I don't think the uniform buffer padding has such kind of limitation.

Reply:
@axinging @fs-eire I checked the spec and our uniform implementation again. The original `maxAlignmentOfField` has no problem, which strictly follows the spec. Currently, all uniform  members are put into a struct. The declaration is like below:
```
struct Uniforms { 
  vec_size:u32,
  aData_shape:vec2<u32>,
  aData_strides:vec2<u32>,
  bData_shape:vec3<u32>,
  bData_strides:vec3<u32>,
  outputData_shape:vec3<u32>, 
  outputData_strides:vec3<u32>
 };
  @group(0) @binding(3) var<uniform> uniforms: Uniforms;
```
From above, we can see that the overall size still needs to follow the alignment and size for struct. The size of the struct is roundUp(AlignOf(S), justPastLastMember), where AlignOf(S) = `maxAlignmentOfField`, justPastLastMember = `currentOffset`. Although 16 won't make our cases failure (the actual buffer length is rounded up to 16), but the meaning is different. If you directly set `maxAlignmentOfField` = 16, it changes the meaning of it since it may be not 16. But if you still think setting `maxAlignmentOfField` = 16 is simpler, please add comment above it to explain why we do it like this in case others feel confused when they see the code here.

Reply:
agree with that "maxAlignmentOfField" is no longer needed.

in the current code, keeping the `const maxAlignmentOfField = 16;` makes it confusing because it does not really mean that. The current usage of this constant is only for padding for the whole Uniform struct. Maybe use a different name?

Reply:
the current name is OK.